.github/workflows/docker.yml

```diff
@@ -27,21 +33,13 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Download required file
-        run: chmod +x ./scripts/download-jar.sh && ./scripts/download-jar.sh
-
       - name: Build Image
         uses: docker/build-push-action@v6
         with:
+          platforms: linux/amd64,linux/arm64
           context: .
-          load: true
-          push: false
-          tags: |
-            ${{ env.IMAGE_NAME }}:test
-            ${{ env.IMAGE_NAME }}:latest
+          push: true
+          tags: ${{ env.IMAGE_NAME }}
 
       - name: Test
-        run: chmod +x ./scripts/test.sh && ./scripts/test.sh ${{ env.IMAGE_NAME }}:test
-
-      - name: Push Image
-        run: docker push ${{ env.IMAGE_NAME }}:latest
+        run: chmod +x ./scripts/test.sh && ./scripts/test.sh ${{ env.IMAGE_NAME }}
```

Comment:
We need to test the build before pushing it to the registry

Reply:
Yeah it's a bit tricky because you run into https://github.com/docker/buildx/issues/59 but let me try something

Reply:
ok take a look at https://github.com/zeta-chain/ton-docker/actions/runs/10819809385/job/30018497194

Reply:
Sometimes, the build might be flaky due to genesis init bugs in JAR's logic. Thus we need to publish the image that **was tested during CI step**.

I see you built a local image and tested it but then built it one more time i.e. published build is not tested

I think we can copy the logic from zeta's `solana-docker` with matrix builds and do simple `docker push ...` as a last step instead of using `docker/build-push-action@v6`

Reply:
I updated `main`. Now the test is `genesis.sh` and it also populates initial blocks. Thus we can use your solution ✅